### ISSUE TITLE
Mantid conda build mysteriously crash analysis cluster nodes

### ISSUE THREAD

**Issue by yxqd** [mantidproject/conda-recipes#15]
It happens when a mcvine simulated nexus file is reduced using SofQW3 algorithm. 

When limiting OMP_NUM_THREADS to 10, it works most of time. (see also: https://github.com/mantidproject/mantid/pull/19369)

**Comment by yxqd** [comment]
The crash was due to incorrect dynamic linking of openblas.

The conda mantid package was built with a later version of numpy from conda-forge, which links against openblas from conda-forge. The mantid libraries in the conda mantid package were linked against the same openblas dynamic library from conda-forge. When the mantid package was installed in the conda environment at the SNS cluster, the numpy package did not get updated to the new numpy that is linked against openblas, and as a result no openblas library was installed in the conda environment at the SNS cluster. Unfortunately, there is a system-wide openblas library installed at /usr/lib64 and it is picked up by the conda mantid library. This mistakenly linked openblas library probably caused problems which lead to crash of the system. The crash stopped when I updated the numpy in the SNS cluster conda environment to the latest one that is linked against openblas library from conda-forge.

I also rebuilt the mantid package by adding openblas explicitly as a dependency. Also numpy was added as an explicit dependency.
